### PR TITLE
Check if datetime is a string

### PR DESCRIPTION
**Pull request by bramkragten**

### CODE REVIEW

**Comment by bramkragten** [comment]
It is used in the Popular entities example of the getting started notebook

File: detective/time.py [hunk 1]
```diff
@@ -41,8 +41,10 @@ def time_category(dtObj):
...
-def sqlalch_datetime(dt_str):
+def sqlalch_datetime(dt):
     """Convert a SQLAlchemy datetime string to a datetime object."""
-    return datetime.strptime(
-        dt_str, '%Y-%m-%d %H:%M:%S.%f'
-    ).replace(tzinfo=UTC)
+    if isinstance(dt, str):
+        return datetime.strptime(
+            dt, '%Y-%m-%d %H:%M:%S.%f'
+        ).replace(tzinfo=UTC)
```
Comment (balloob) [inline detective/time.py]: Can you check what the `tzinfo` is from datetime objects returned from Postgres? I wouldn't be surprised if they are not timezone aware, in which case we should still do the `replace` on it.

Comment (bramkragten) [reply detective/time.py]:
```python
    print(event.time_fired)
    print(event.time_fired.tzinfo)
```

```
2018-11-19 19:20:53.847911+01:00
psycopg2.tz.FixedOffsetTimezone(offset=60, name=None)
```

Comment (balloob) [reply detective/time.py]: Okay, so it seems that your returned date is already localized. In that case we should not replace tzinfo, but we should convert it to utc, to normalize the datetimes returned from the DB. Else we can't write generic notebooks on top of it.

File: detective/time.py [hunk 2]
```diff
@@ -49,4 +49,4 @@ def sqlalch_datetime(dt):
         ).replace(tzinfo=UTC)
     if dt.tzinfo is not None and dt.tzinfo.utcoffset(dt) is not None:
         return dt.astimezone(UTC)
-    return dt.replace(tzinfo=pytz.UTC)
+    return dt.replace(tzinfo=UTC)
```
Comment (bramkragten) [inline detective/time.py]: Not sure, should this be `localize()` or `replace()`?

**Pull request merged**